homeassistant/components/balboa/light.py

```diff
@@ -0,0 +1,54 @@
+"""Support for Balboa Spa lights."""
+from __future__ import annotations
+
+from typing import Any, cast
+
+from pybalboa import SpaClient, SpaControl
+from pybalboa.enums import OffOnState, UnknownState
+
+from homeassistant.components.light import LightEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .entity import BalboaEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the spa's lights."""
+    spa: SpaClient = hass.data[DOMAIN][entry.entry_id]
+    async_add_entities(BalboaLightEntity(control) for control in spa.lights)
+
+
+class BalboaLightEntity(BalboaEntity, LightEntity):
+    """Representation of a Balboa Spa light entity."""
+
```

Comment:
It is required to set the supported color modes (https://developers.home-assistant.io/blog/2024/02/12/light-color-mode-mandatory/)

```suggestion
class BalboaLightEntity(BalboaEntity, LightEntity):
    """Representation of a Balboa Spa light entity."""

    _attr_color_mode = ColorMode.ONOFF
    _attr_supported_color_modes = {ColorMode.ONOFF}

```